File: php/1038.php
<?php
    $e = fgets(STDIN);
    $e = explode(' ', $e);
    if ($e[0] == 1){
        $e[0] = 4.0;
    }
    else if ($e[0] == 2){
        $e[0] = 4.5;
    }
    else if ($e[0] == 3){
        $e[0] = 5.0;
    }
    else if ($e[0] == 4){
        $e[0] = 2;
    }
    else{
        $e[0] = 1.5;
    }
    
    printf("Total: R$ %.2f\n", $e[0]*$e[1]);
?>

File: c/1038.c
#include <stdio.h>

int main(){
    double e, e1;
    scanf("%lf %lf", &e, &e1);

    if (e == 1){
        e = 4.0;
    }
    else if (e == 2){
        e = 4.5;
    }
    else if (e == 3){
        e = 5.0;
    }
    else if (e == 4){
        e = 2;
    }
    else{
        e = 1.5;
    }
    
    printf("Total: R$ %.2f\n", e*e1);
}